db/seed.rb

```diff
@@ -29,7 +29,7 @@ def import_taxonomy!(values_data:, attributes_data:, verticals_data:)
 
       vputs("  → Attributes and values")
       Attribute.insert_all_from_data(attributes_data["base_attributes"])
-      Value.insert_all_from_data(values_data)
+      Value.insert_all_from_data(values_data, attributes_data["base_attributes"])
```

Comment:
I wonder if it would be clearer to write it like this.

```suggestion
    attributes_data["base_attributes"].each do |attribute|
      Value.insert_all_from_data(attribute.values, custom_sort: attribute[:custom_sorting])
    end
```

Then above 

```
  def insert_all_from_data(data, custom_sort)
     data.each_with_index { |value, index| value[:position] = index+1 } if custom_sort

      insert_all!(Array(data))
  end
```

Reply:
Attribute values only stores the friendly_id of each value. The rest of the fields need to come from that values_data array